vue-components/src/components/Button.vue

```diff
@@ -112,6 +149,32 @@ $base: '.wikit-Button';
 			}
 		}
 
+		&#{$base}--quiet--primaryProgressive {
```

Comment:
```suggestion
		&#{$base}--quiet--progressive {
```

Reply:
The reason I went with primaryProgressive was https://github.com/wmde/wikit/pull/324/files#diff-87e4cc0a282e4a634e399ca54e31d3afab36d683f67774e79dd48aec552d4c68L30
> // TODO why does the shorter version cause problems with doc-gen?

I should investigate why this happens.

Reply:
@Ladsgroup thank link doesn't seem to work?

Also the comment

> // TODO why does the shorter version cause problems with doc-gen?

has nothing to do with the prop, but refers to the validator function. At some point we (Pablo and I at the time) tried to proof that we paid attention in functional programming class and do

```js
			validator: Array.prototype.includes.bind( [ 'neutral', 'primaryProgressive', 'primaryDestructive' ] ),
```
instead of
```js
			validator( value: string ): boolean {
				return [ 'neutral', 'primaryProgressive', 'primaryDestructive' ].includes( value );
			},
```

which should be equivalent in theory, but exploded in the build step. I don't think this was a good idea to begin with and now the comment is confusing and should probably just be removed, sorry.

Reply:
> @Ladsgroup that link doesn't seem to work?

It's just the link to the comment.

I see, I completely misunderstood it.

Reply:
Yeah, it took me a minute to understand it too, and I'm pretty sure I wrote it. Removed in https://github.com/wmde/wikit/pull/327